ceph-setup/build/build

```diff
@@ -31,17 +31,8 @@ rm -rf release
 echo "Running submodule update ..."
 git submodule update --init
 
-# Flavor Builds support
-
-if [ "${FLAVOR}" == "notcmalloc" ]
-then
-    echo "Detected notcmalloc flavor: will use flag: -DALLOCATOR=libc"
-    CEPH_EXTRA_RPMBUILD_ARGS="--without tcmalloc"
-    CEPH_EXTRA_CMAKE_ARGS="$CEPH_EXTRA_CMAKE_ARGS -DALLOCATOR=libc -DWITH_BOOST_VALGRIND=ON"
-else
-    CEPH_EXTRA_RPMBUILD_ARGS="--with tcmalloc"
-    CEPH_EXTRA_CMAKE_ARGS="$CEPH_EXTRA_CMAKE_ARGS -DALLOCATOR=tcmalloc"
-fi
+CEPH_EXTRA_RPMBUILD_ARGS="--with tcmalloc"
+CEPH_EXTRA_CMAKE_ARGS="$CEPH_EXTRA_CMAKE_ARGS -DALLOCATOR=tcmalloc"
```

Comment:
this is probably unnecessary.. mostly it means that if the distro or build-requires don't have tcmalloc installed then the build will fail installed of falling back to libc.